app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php

```diff
@@ -210,7 +188,6 @@ protected function parseSelections($rowData, $entityId)
                 }
                 $this->_cachedSkus[] = $option['sku'];
                 if (!isset($this->_cachedOptions[$entityId][$option['name']])) {
-                    $this->_cachedOptions[$entityId][$option['name']] = [];
```

Comment:
Hello @engcom-Charlie, thank you for your contribution and cleanup. 

Just to make sure I understand, this line seems the only change to business logic but since in the following it is overwritten I don't get how the removal of this line affects the issue.

Maybe I'm missing something else.

Let me know, thanks!

Reply:
Hello @aleron75, the changes You are talking about are just code clean-up. Please see https://github.com/magento/magento2/blob/6fdcc885b25603810a8a077a47cf4c27e3f274de/app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php#L656.

Thank You.